src/components/nano/ModalChangeAddress.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect, useRef, useState } from 'react';
+import { t } from 'ttag';
+import $ from 'jquery';
+import PropTypes from 'prop-types';
+import AddressList from '../../components/AddressList';
+import { CHANGE_ADDRESS_LIST_COUNT } from '../../constants';
+import { connect } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import { editAddressNC } from '../../actions/index';
+import { getGlobalWallet } from '../../modules/wallet';
+
+/**
+ * Component that shows a modal to change the address of a Nano Contract
+ *
+ * @memberof Components
+ */
+function ModalChangeAddress(props) {
```

Comment:
suggestion: Explicit props also on method declaration.

```suggestion
function ModalChangeAddress({ nanoContractId, onClose }) {
```

Reply:
Done, thanks! https://github.com/HathorNetwork/hathor-wallet/pull/623/commits/bf9a293d902c1c9062f3871652e358aa8797ab48